sovrin_client/test/agent/acme.py

```diff
@@ -1,151 +1,154 @@
+                       salary_bracket="between $50,000 to $100,000")
+
+    backend.add_record(2,
+                       first_name="Carol",
+                       last_name="Atkinson",
+                       employee_status="Permanent",
+                       experience="2 years",
+                       salary_bracket="between $60,000 to $90,000")
+
+    backend.add_record(3,
+                       first_name="Frank",
+                       last_name="Jeffrey",
+                       employee_status="Temporary",
+                       experience="4 years",
+                       salary_bracket="between $40,000 to $80,000")
+
+    backend.add_record(4,
+                       first_name="Bob",
+                       last_name="Richards",
+                       employee_status="On Contract",
+                       experience="3 years",
+                       salary_bracket="between $50,000 to $70,000")
+
+    agent.set_issuer_backend(backend)
+
+    return agent
+
+
+async def bootstrap_acme(agent):
+    agent._proofRequestsSchema = {
+        "Job-Application-v0.2": {
+            "name": "Job-Application",
+            "version": "0.2",
+            "attributes": {
+                "first_name": "string",
+                "last_name": "string",
+                "phone_number": "string",
+                "degree": "string",
+                "status": "string",
+                "ssn": "string"
+            },
+            "verifiableAttributes": ["degree", "status", "ssn"]
+        }
+    }
+    await bootstrap_schema(agent,
+                           'Job-Certificate',
+                           'Job-Certificate',
+                           '0.2',
+                           primes["prime1"][0],
+                           primes["prime1"][1])
+
+    await bootstrap_schema(agent,
+                           'Job-Application',
+                           'Job-Application',
+                           '0.2',
+                           primes["prime2"][0],
+                           primes["prime2"][1])
 
 
-def createAcme(name=None, wallet=None, basedirpath=None, port=None):
-    return createAgent(AcmeAgent, name=name or "Acme Corp",
-                       wallet=wallet or buildAcmeWallet(),
-                       basedirpath=basedirpath, port=port, clientClass=TestClient)
+if __name__ == "__main__":
+    args = RunnableAgent.parser_cmd_args()
 
+    port = args[0]
+    if port is None:
+        port = 6666
+    agent = create_acme(name='Acme Corp', wallet=buildAcmeWallet(),
+                        base_dir_path=None, port=port)
+    RunnableAgent.run_agent(agent, bootstrap=bootstrap_acme(agent))
 
-if __name__ == "__main__":
-    port=6666
-    TestWalletedAgent.createAndRunAgent(
-        name="Acme Corp",  agentClass=AcmeAgent, wallet=buildAcmeWallet(),
-        basedirpath=None, port=port, looper=None, clientClass=TestClient,
-        cliAgentCreator=lambda: createAcme(port=port))
```

Comment:
This parameter (cliAgentCreator=lambda: createAcme(port=port)) was passed to support agent's to be run in cli mode. So say from command line, if I type:
**python faber.py --withcli**, then faber agent will start in cli mode. So that won't work now I guess.